.github/workflows/daily.yaml

```diff
@@ -53,6 +53,11 @@ jobs:
             markers: 'daily and (remote or not remote) and not gpu and not vision and not doctest'
             pytest_command: 'coverage run -m pytest'
             composer_package_name: 'composer'
+          - name: 'daily-cpu-3.10-2.1'
```

Comment:
this file also needs the normal case (in pr-cpu) at the top

Reply:
good catch, added 👍